pkg/operator/starter.go

```diff
@@ -39,15 +53,22 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		return err
 	}
 
-	kubeInformers := v1helpers.NewKubeInformersForNamespaces(kubeClient, util.TargetNamespace, util.OperatorNamespace, util.UserSpecifiedGlobalConfigNamespace)
+	kubeInformers := v1helpers.NewKubeInformersForNamespaces(kubeClient,
+		"",
```

Comment:
let's put a comment in code to indicate this means cluster scoped, as you clarified in the PR comment https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/216/commits/09eb355f0d0a7866c17b28f5a340936a7326eeb5#r644180187

Reply:
Or go with https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/216#discussion_r644215148 instead

Reply:
I'll add a comment to say "empty means cluster-scoped"

Reply:
sounds good thanks

Reply:
updated